### PR TITLE
fix: Remove Z-Index Requirement for Listing Tags

### PR DESCRIPTION
**Pull request by chadbrokaw-exygy**
# Pull Request Template

#29 

## Description

DAHLIA identified an issue where the listing tag renders on top of the slide out mobile menu. This occurred whenever the listing tag was present.

## How Can This Be Tested/Reviewed?

**In storybook**
Check to ensure that the listing tags are rendering correctly.
**In DAHLIA (or Bloom)**
1. Check to ensure that the listing tags render correctly on the listing details page and the rental listings page.
2. Using the inspector or an emulator, change the viewport to be a mobile version. 
3. Go to the listings page and find a listing with listing tags on them. 
4. Open the slide out menu by clicking the hamburger icon in the top right
5. The Listing tag should be underneath the menu
6. Try this on the listing details page

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have made any corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added or updated stories if new changes are not captured in Storybook
- [x] New and existing unit tests pass locally with my changes
- [ ] I have exported any new pieces added to ui-components
- [ ] I have documented this change in the changelog, and any breaking changes are well described

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Either review the Storybook preview or pull the changes down locally and test that the acceptance criteria is met
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

On merge, squash commits.

### CODE REVIEW

File: src/blocks/ImageCard.tsx [hunk 1]
```diff
@@ -174,6 +156,24 @@ const ImageCard = (props: ImageCardProps) => {
           )}
         </figure>
         {getStatuses()}
+        <div className="image-card-tag__wrapper">
```
Comment (ludtkemorgan) [inline src/blocks/ImageCard.tsx:159]: One thing that this change does is it makes the screen reader read the tags last. So for this example the "application due date" is read before the "Seniors 62+". 
image

I don't think this is an issue but @ColinBuyck & @emilyjablonski do you have any opinions on this?

Comment (emilyjablonski) [reply src/blocks/ImageCard.tsx:159]: That seems probably preferable, I'll link to core and test out as well

Comment (chadbrokaw-exygy) [reply src/blocks/ImageCard.tsx:159]: Thank you for calling that out @ludtkemorgan!

**Review by emilyjablonski — APPROVED** [review]
A great improvement I think in screen reader UX actually!

**Review by ludtkemorgan — APPROVED** [review]
🍇

**Pull request merged**